util/crops.go

```diff
@@ -59,7 +59,7 @@ func NewCrop(name string) *Crop {
 // Otherwise it just returns false
 func (c *Crop) Revert() bool {
 	c.stage = c.revertTo
-	if c.revertTo < 0 {
+	if c.revertTo == 0 {
```

Comment:
Check https://github.com/bcvery1/Deadication/blob/master/util/crops.go#L37 "Apple trees" revert to stage 3, not 0.  This line will not work.  We actually want to check if `c.revertTo > -1` (originally it was `<`)